class/Defaults/Trigger/WordPress/ActivePlugin.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * WordPress Active Plugin trigger
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\Trigger\WordPress;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+use BracketSpace\Notification\Abstracts;
+
+/**
+ * Active Plugin trigger class
+ */
+class ActivePlugin extends Abstracts\Trigger{
+
+	/**
+	 * Constructor
+	 */
+	public function __construct(){
+
+		parent::__construct( 'wordpress/active_plugin',  __( 'Plugin Active', 'notification' ) );
+
+		$this->add_action( 'activated_plugin', 10, 2 );
+		$this->set_group( __( 'WordPress', 'notification' ) );
+		$this->set_description( __( 'Fires when plugin is active', 'notification' ) );
+
+	}
+
+	/**
+	 * Gets specific update type
+	 *
+	 * @param  string $plugin PluginPath.
+	 * @return void
+	 */
+	public function action($plugin)
+	{
+		$this->plugin = $plugin;
+	}
+
+	/**
+	 * Registers attached merge tags
+	 *
+	 * @return void
+	 */
+	public function merge_tags()
```

Comment:
As you may see, both triggers use the same merge tags and you are violating the DRY rule. Would be much much better to create separate MergeTag classes for each of these properties and create a parent class which will add all of them at once. Please take a look at Merge Tags for Posts and how the PostTrigger class look like